lib/main_containers.pm

```diff
@@ -155,6 +155,24 @@ sub load_host_tests_helm {
     }
 }
 
+sub load_host_tests_in_k8s {
+    my ($run_args) = @_;
+    my $backends = undef;
+
+    # if (is_sle('15-sp3+')) {
+    $backends = get_var("K8S_BACKEND", "GKE,EKS,AKS");
+    # } else {
+    #     die("KS backend not supported on this host");
+    # }
+
+    foreach (split(',\s*', $backends)) {
+        push @{$run_args->{backends}}, $_;
+        loadtest('containers/push_container_image_to_pc', run_args => $run_args, name => "push_container_image_to_pc_" . $_);
```

Comment:
```suggestion
        loadtest('containers/push_container_image_to_pc', run_args => $run_args, name => "push_container_image_to_" . $_);
```

Reply:
yes, I was also thinking that `..._to_pc` sounds a bit strange, better to use the provider name.

Reply:
done

Reply:
did you change the name in the end?